types/algoliasearch/lite/index.d.ts

```diff
@@ -101,6 +101,11 @@ declare namespace algoliasearch {
       attributes: string[],
       cb: (err: Error, res: {}) => void
     ): void;
+    /**
+     * Gets specific attributes from an object
+     * https://github.com/algolia/algoliasearch-client-js#find-by-ids---getobjects
```

Comment:
This is a dead link
I believe something like that would be better:
https://www.algolia.com/doc/api-reference/api-methods/get-objects/#retrieve-only-one-object

Reply:
Updated the links 🙂

Reply:
Thanks a lot 🙏 ❤️